config/locales/mailers/en.yml

```diff
@@ -34,6 +34,10 @@ en:
         header: "A new request submitted by %{requester_name} needs your %{step_type_noun}"
         cta: "Add comment"
     proposal_mailer:
+      generic_approval_name: 
```

Comment:
would it be possible to make these dynamic now? We could do this via a decorator for each user based on their role. 

Could also keep it mocked out but if we do we need to create a separate card so we don't forget to switch these out. These roles are accurate for some NCR requests but not all (eg: not all will have 3 approvers, not all approval chains will have these roles)